lib/bitcoin/BitcoinClient.ts

```diff
@@ -570,36 +570,36 @@ export default class BitcoinClient {
 
   private async createSpendToFreezeTransaction (
     previousFreezeTransaction: BitcoreTransactionWrapper,
-    previousFreezeUntilBlock: number,
-    freezeUntilBlock: number): Promise<[Transaction, string]> {
+    previousFreezeDurationInBlocks: number,
+    newFreezeDurationInBlocks: number): Promise<[Transaction, string]> {
 
     // tslint:disable-next-line: max-line-length
-    console.info(`Creating a freeze transaction with freeze until block: ${freezeUntilBlock} from previously frozen transaction with id: ${previousFreezeTransaction.id}`);
+    console.info(`Creating a freeze transaction with freeze time in blocks: ${newFreezeDurationInBlocks} from previously frozen transaction with id: ${previousFreezeTransaction.id}`);
 
-    const freezeScript = BitcoinClient.createFreezeScript(freezeUntilBlock, this.bitcoinWallet.getAddress());
+    const freezeScript = BitcoinClient.createFreezeScript(newFreezeDurationInBlocks, this.bitcoinWallet.getAddress());
     const payToScriptHashOutput = Script.buildScriptHashOut(freezeScript);
     const payToScriptAddress = new Address(payToScriptHashOutput);
 
     // We are creating a spend transaction and are paying to another freeze script.
     // So essentially we are re-freezing ...
     const reFreezeTransaction = await this.createSpendTransactionFromFrozenTransaction(
       previousFreezeTransaction,
-      previousFreezeUntilBlock,
+      previousFreezeDurationInBlocks,
       payToScriptAddress);
 
     return [reFreezeTransaction, freezeScript.toHex()];
   }
 
   private async createSpendToWalletTransaction (

```

Comment:
Nit: I probably mentioned this earlier and forgot the reason you gave already (was it for testing?). Looks like it just a method wrapping another method.

Reply:
`createSpendTransactionFromFrozenTransaction` is a utility function and gets called by 2 functions: `createSpendToFreezeTransaction` & `createSpendToWalletTransaction`. The 2nd one can be removed as it is literally one line calling the utility function but want to leave this for another opportunity to get this change out (there're test changes as well that I want to avoid).